.github/support.md

```diff
@@ -0,0 +1,27 @@
+## Are you having trouble getting started?
+If you haven't already, please checkout our [documentation](https://docs.bugsnag.com/performance/integration-guides/go/) for guides, references and tutorials.
+
+Or, if you wish you can [contact us directly](mailto:support@bugsnag.com) for assistance on integrating Bugsnag into your application, troubleshooting an issue or a question about our supported features.
+
+When contacting support, please include as much information as necessary, including:
+
+- example code snippet
+- steps to reproduce
+- expected/actual behaviour 
+
+* Bugsnag Performance library version:
+* Bugsnag Error Monitoring library version (if any):
+* Go version:
+* Integration framework version:
+    * Martini:
+    * Negroni:
+    * net/http:
+    * Revel:
+    * Other:
+
+## Bug or Feature Requests
+If you would like to raise a bug or feature request please do so by creating a [New Issue](https://github.com/bugsnag/bugsnag-go-performance/issues/new/choose) and selecting bug or feature.
+Please note: we cannot promise that we will fulfil all requests
+
+## Pull Requests
+If you have made a fix and would like to raise a pull request, please read our [CONTRIBUTING.md](../CONTRIBUTING.md) file before creating the pull request.
```

Comment:
`CONTRIBUTING.md` doesn't exist, and therefore the link is broken. This isn't included in the templates we push, so not sure if this is something that should be added by Platforms..? Is this still something we should be including?